java/src/lib.rs

```diff
@@ -103,8 +111,11 @@ pub extern "system" fn Java_glide_ffi_resolvers_RedisValueResolver_valueFromPoin
     redis_value_to_java(&mut env, *value)
 }
 
+/// # Safety
+///
+/// * Function throws a checked java exception if fails to get the socket path.
```

Comment:
Does this actually lead to UB in any cases?

Reply:
No, but linter requires me to add a safety section

Reply:
That's because you marked the function as unsafe. If it really needs to be unsafe, then it should be marked unsafe. Otherwise, neither the unsafe nor the comment are necessary.

Reply:
I agree with that.
There is an ongoing discussion to enforce `unsafe` for all `no_mangle` functions: https://github.com/rust-lang/rust/issues/28179. So I added it here, but ready to revert.

Reply:
Okay, but that's not the documented safety condition. If you need to add unsafe to a function, you need to document what the caller needs to do to safely call the function.

Reply:
Fixed